File: sb_restaurant_reservation_rest/src/main/java/dk/sb_restaurant_reservation_rest/model/Reservation.java
package dk.sb_restaurant_reservation_rest.model;

import java.time.LocalDate;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Table(name="reservations")
public class Reservation {

	@Id
	@Column(value="id")
	private Integer id;
	
	@Column(value="date")
	private LocalDate date;
	
	@Column(value="hour_from")
	private Integer from;
	
	@Column(value="name")
	private String name;
	
	@Column(value="email")
	private String email;
	
	@Column(value="phone")
	private String phone;
	
	@Column(value="restaurant_id")
	private Integer resId;
	
	@Column(value="reserved_seats")
	private Integer resSeats;

	public Reservation(Integer id, LocalDate date, Integer from, String name, String email, String phone, Integer resId,
			Integer resSeats) {
		super();
		this.id = id;
		this.date = date;
		this.from = from;
		this.name = name;
		this.email = email;
		this.phone = phone;
		this.resId = resId;
		this.resSeats = resSeats;
	}

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public LocalDate getDate() {
		return date;
	}

	public void setDate(LocalDate date) {
		this.date = date;
	}

	public Integer getFrom() {
		return from;
	}

	public void setFrom(Integer from) {
		this.from = from;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getPhone() {
		return phone;
	}

	public void setPhone(String phone) {
		this.phone = phone;
	}

	public Integer getResId() {
		return resId;
	}

	public void setResId(Integer resId) {
		this.resId = resId;
	}

	public Integer getResSeats() {
		return resSeats;
	}

	public void setResSeats(Integer resSeats) {
		this.resSeats = resSeats;
	}
	
}

File: Ng_RestaurantReservation/src/app/model/reservation.ts
export class Reservation {

    private id: number;
    private name: string;
	private email: string;
	private phone: string;
    private date: string;
    private seats: number;
    private slot: number;

     public constructor() {
        this.id = 0;
        this.name = "";
        this.email = "";
        this.phone = "";
        this.date = "";
        this.seats = 0;
        this.slot = 0;

    }

    public setId(id: number) {
        this.id = id;
    }

    public getId(): number {
        return this.id;
    }

    public setName(name: string) {
        this.name = name;
    }

    public getName(): string {
        return this.name;
    }

    public setEmail(email: string) {
        this.email = email;
    }

    public getEmail(): string {
        return this.email;
    }

    public setPhone(phone: string) {
        this.phone = phone;
    }

    public getPhone(): string {
        return this.phone;
    }

     public setDate(date: string) {
        this.date = date;
    }

    public getDate(): string {
        return this.date;
    }

    public setSeats(seats: number) {
        this.seats = seats;
    }

    public getSeats(): number {
        return this.seats;
    }

    public setSlot(slot: number) {
        this.slot = slot;
    }

    public getSlot(): number {
        return this.slot;
    }
}
